scripts/report_scripts/variantreport_p_sample.Rmd

```diff
@@ -337,6 +337,9 @@ muts <- c(match.df$AA_mut)
 # get bulk frequency values
 bulk <- as.numeric(match.df$freq)
 
+ifelse (is.null(dim(muts)),
+  executeDeconvolution <- as.logical("FALSE"), 
+  executeDeconvolution <- as.logical("TRUE" ))
```

Comment:
This is weird.  Why `as.logical("TRUE")` instead of just `TRUE`?

I'd write the whole thing like this:

`executeDeconvolution <- !is.null(dim(muts))`